tasks/config/tasks.js

```diff
@@ -40,12 +40,12 @@ module.exports = {
     },
 
     filenames: {
-        concatenatedCSS: 'semantic.css',
-        concatenatedJS: 'semantic.js',
-        concatenatedMinifiedCSS: 'semantic.min.css',
-        concatenatedMinifiedJS: 'semantic.min.js',
-        concatenatedRTLCSS: 'semantic.rtl.css',
-        concatenatedMinifiedRTLCSS: 'semantic.rtl.min.css',
+        concatenatedCSS: 'semantic' + release.versionInFileName + '.css',
+        concatenatedJS: 'semantic' + release.versionInFileName + '.js',
+        concatenatedMinifiedCSS: 'semantic' + release.versionInFileName + '.min.css',
+        concatenatedMinifiedJS: 'semantic' + release.versionInFileName + '.min.js',
+        concatenatedRTLCSS: 'semantic.rtl' + release.versionInFileName + '.css',
+        concatenatedMinifiedRTLCSS: 'semantic.rtl' + release.versionInFileName + '.min.css',
```

Comment:
We should make this consistens and always put the version directly after "semantic"

```suggestion
        concatenatedRTLCSS: 'semantic' + release.versionInFileName + '.rtl.css',
        concatenatedMinifiedRTLCSS: 'semantic' + release.versionInFileName + '.rtl.min.css',
```